src/main/server/map/server-map-component.ts

```diff
@@ -150,4 +166,14 @@ export class ServerMapComponent extends SharedMapComponent {
       // https://stackoverflow.com/questions/17945972/converting-rgba-values-into-one-integer-in-javascript
       return ((pixelValue & 0xff000000) >>> 24) / 255;
    }
+
+   private getNrOfMoonPixel(): number {
```

Comment:
What is NR?

Reply:
do you mean number?

If so, `Pixel` should be `Pixels`, and Nr should be number

`getNumberOfMoonPixels` or `getMoonPixelCount`

Reply:
Fixed